.gitignore

```diff
@@ -2,3 +2,4 @@ dist/
 .idea/
 VERSION
 .tmp/
+*.swp
```

Comment:
what's this?

Reply:
"swp" files are temporary files generated by the vim.